api/package.json

```diff
@@ -13,7 +13,6 @@
     "drizzle:studio": "drizzle-kit studio"
   },
   "dependencies": {
-    "cookie-parser": "^1.4.6",
```

Comment:
Removed our reliance on cookie, as using query parameters makes it cacheable with [Vercel's Edge Network](https://vercel.com/docs/edge-network/caching).